### PR TITLE
Update nanopb py3 0.3.9.5 -> 0.3.9.6

### PR DESCRIPTION
**Pull request by vimanyu**
This fixes issues building firestore SDK with Python3.
Interestingly, latest version nanopb-0.4.2 still errors with Python3 but 0.3.9.6 works.
The git commit for this fix in nanopb
nanopb/nanopb@63f8e5e

PR for nanopb-podspec,
https://github.com/google/nanopb-podspec/pull/14

SpecStaging repo is already updated with the new nanopb pod (1.30906.0).
https://github.com/firebase/SpecsStaging

Hey there! So you want to contribute to a Firebase SDK?
Before you file this pull request, please read these guidelines:

Discussion
Users ran into issues when using Python3 for building firestore. nanopb 0.3.9.6 has a fix for it. 
firebase/firebase-cpp-sdk#48

Testing
Make sure all existing tests in the repository pass after your change.
If you fixed a bug or added a feature, add a new test to cover your code.
API Changes
At this time we cannot accept changes that affect the public API. If you'd like to help
us make Firebase APIs better, please propose your change in a feature request so that we
can discuss it together.

### CODE REVIEW

**Comment by paulb777** [comment]
All of the other pods using nanopb also have generated source and should do a similar update/confirmation that the only change is the version comment.

The CI failures will require staging a version of FirebaseAnalytics and its GoogleAppMeasurement dependency on SpecsStaging. The shortest path is to take the published versions in https://github.com/CocoaPods/Specs/blob/master/Specs/e/2/1/FirebaseAnalytics/6.7.0/FirebaseAnalytics.podspec.json, update the nanopb dep and push them there.

**Comment by paulb777** [comment]
Please add a link to a nanopb-podspec PR in the description

File: GoogleDataTransport/GDTWatchOSTestApp/Podfile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+source 'https://github.com/firebase/SpecsStaging.git'
```
Comment (wilhuff) [inline GoogleDataTransport/GDTWatchOSTestApp/Podfile:1]: This shouldn't be committed, should it?

Comment (paulb777) [reply GoogleDataTransport/GDTWatchOSTestApp/Podfile:1]: Yes.  It should be in all this repo's Podfiles to enable pre-release testing of dependencies.

**Comment by vimanyu** [comment]
Thanks @wilhuff and @paulb777! Merged to master!

**Pull request merged**